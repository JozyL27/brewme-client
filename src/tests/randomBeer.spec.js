import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import RandomBeer from '../components/randomBeer/randomBeer'


describe(`random beer component`, () => {
    it(`random beer component renders without crashing`, () => {
        const div = document.createElement('div')
        ReactDOM.render(<RandomBeer />, div)
        ReactDOM.unmountComponentAtNode(div)
    })


    it(`renders the UI as expected`, () => {
        const tree = renderer
            .create(<RandomBeer />)
            .toJSON()
            expect(tree).toMatchSnapshot()
    })
})