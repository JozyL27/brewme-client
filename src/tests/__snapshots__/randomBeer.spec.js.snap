// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`random beer component renders the UI as expected 1`] = `
<section
  className="random"
>
  <button
    className="randButton"
    onClick={[Function]}
  >
    Get Random Beer!
  </button>
</section>
`;
